src/main/java/com/fvv/bookstore/util/DateUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.fvv.bookstore.util;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Class DateUtil to convert a String into a Date
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 30 de mar de 2018</p>	
+ * @version 1.0 	
+ *
+ */
+public final class DateUtil {
```

Comment:
Add a method that gets a Date and return a String formatted in the proper mask, then update the Magazine.toString() method to call this new method, so in the list output a proper mask can be exhibited.

Reply:
Ok